### DISCUSSION TITLE
Performance mode restriction over guenine power adaptator plug in detection

### DISCUSSION THREAD

**Discussion by Helium100** [seerge/g-helper#1229] [Q&A] [answered]
Hi,
In Armoury Crate, the "Turbo mode" and "Manual mode" is only available when the guenine alimentation is plugged in.
It does not worked when powered through USB-C hub's Power Delivery (even if PD is delivering the same amount of power)
Is there such limitation with G-helper?
Thanks

**Comment by seerge** [comment, marked as answer]
@Helium100 you can use Turbo mode on usb-c. However same as any other mode - it will be different comparing to what laptop can do when plugged in to barrel charger. All modes are hadcoded in BIOS

**Reply by Helium100** [reply]
Okay, thanks! I will likely give it a try to see what is the behavior. Anyway if the turbo mode is indeed available in usb-c, then it's a huge advantage for g-helper compared to Armoury Crate that just keep blocking these modes for no relevant reason.

**Reply by Helium100** [reply]
Okay so, I try to use g-helper to see what's the behavior with the Hub USB-c powering the PC intead of the guenine charger,
And, well, it's awesome no more limitations! I can even switch to "Turbo mode" with no charger plugged at all!

I thing that might deserve a mention into the "Feature section", and perhaps also into "Main Advantages" because I am definitely going to use g-helper thanks to this feature,
thank you!
